src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/LiteHash.Apple.cs

```diff
@@ -21,6 +21,27 @@ internal static LiteHmac CreateHmac(string hashAlgorithmId, ReadOnlySpan<byte> k
             PAL_HashAlgorithm algorithm = HashAlgorithmNames.HashAlgorithmToPal(hashAlgorithmId);
             return new LiteHmac(algorithm, key, preinitialize: true);
         }
+
+        internal static LiteXof CreateXof(string hashAlgorithmId)
+        {
+            _ = hashAlgorithmId;
+            throw new PlatformNotSupportedException();
+        }
+    }
+
+    internal struct LiteXof : ILiteHash
```

Comment:
Since the "real" one is readonly, should this one and the one below also be readonly?

Reply:
(although I see in reading further in the PR that there are multiple "real" ones, only some of which are readonly)